AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/EventAdminController.cs

```diff
@@ -73,9 +73,11 @@ public EventController(IImageService imageService, IMediator mediator, IValidate
         public async Task<IActionResult> Create(int campaignId)
         {
             var campaign = await _mediator.SendAsync(new CampaignSummaryQuery { CampaignId = campaignId });
-            if (campaign == null || !User.IsOrganizationAdmin(campaign.OrganizationId))
+
+            var authorizableCampaign = await _mediator.SendAsync(new AuthorizableCampaignQuery(campaign.Id, campaign.OrganizationId));
+            if (authorizableCampaign == null || !await authorizableCampaign.UserCanManageChildObjects())

```

Comment:
You shouldn't have to do the null check on authorizableCampaign here since we return a NotFound (null object) authorizable campaign when we can't locate it, which no-ops to always return false on the implemented methods.